WindowsServerDocs/storage/refs/refs-overview.md

```diff
@@ -85,7 +86,7 @@ Deploying ReFS on basic disks is best suited for applications that implement the
 
 > [!IMPORTANT]
 > If you plan to use ReFS for CSV (Cluster Shared Volumes), please consider the limitations to pre-format your later CSV volumes with ReFS.
-> For CSV: NTFS should be used for traditional SANs. ReFS should be used on top of S2D.
+> For CSV: NTFS should be used for traditional SANs. ReFS should be used on top of S2D. Refer chapter functionality.
```

Comment:
Readability improvement suggestion:
```suggestion
> For CSV: NTFS should be used for traditional SANs. ReFS should be used on top of S2D. Refer to the functionality chapter.
```
A secondary phrasing could be: "Refer to the manual chapter concerning functionality."